stdlib/jwt/src/main/ballerina/src/jwt/jwt_validator.bal

```diff
@@ -47,6 +48,7 @@ public type JwtTrustStoreConfig record {|
     string certificateAlias;
 |};
 
+// TODO: Remove since not used anymore.
```

Comment:
Why is TODO here?

Reply:
Its no longer used. But since this is public record, didn't remove it. Is that okay to remove this?

Reply:
No, we can't remove it. instead of `TODO`. Shall we a proper comment. describe why it is not used

Reply:
Updated with https://github.com/ballerina-platform/ballerina-lang/pull/24195/commits/817c3fed4978ed4038386599ae0d15eb486cd956